eotsmanager/cmd/eotsd/daemon/keys.go

```diff
@@ -33,8 +33,45 @@ func NewKeysCmd() *cobra.Command {
 
 	// Find the "add" subcommand
 	addCmd := util.GetSubCommand(keysCmd, "add")
-	if addCmd == nil {
```

Comment:
we should panic if the command is not found, as it should always be found to modify it